code/game/machinery/launch_pad.dm

```diff
@@ -180,47 +179,37 @@
 		usr.visible_message("<span class='notice'>[usr] starts closing [src]...</span>", "<span class='notice'>You start closing [src]...</span>")
 		if(do_after(usr, 30, target = usr))
 			usr.put_in_hands(briefcase)
-			forceMove(briefcase)
+			moveToNullspace() //hides it from suitcase contents
 			closed = TRUE
 
 /obj/machinery/launchpad/briefcase/attackby(obj/item/I, mob/user, params)
 	if(istype(I, /obj/item/launchpad_remote))
 		var/obj/item/launchpad_remote/L = I
+		if(L.pad == src) //do not attempt to link when already linked
```

Comment:
@SpaceManiac That fixed the issue, awesome. 
One issue I found is that hitting the remote's "remove" button unliks it and sets `pad = null`, but this check treats it as if it was still linked.